karma.conf.js

```diff
@@ -1,3 +1,6 @@
+var isparta = require('isparta');
+var istanbul = require('browserify-istanbul');
```

Comment:
**istanbul** is a very popular code-coverage reporter. And **isparta** is for ES6. Both are provinces. 

Amazing name for the package though.